data/scripts/molgenis-model-update/requirements.txt

```diff
@@ -1,12 +1,14 @@
 certifi==2023.7.22
 charset-normalizer==2.1.1
 idna==3.4
-numpy==1.26.2
-pandas==2.1.2
+numpy
```

Comment:
huh, you always want to use latest? I am okay with this but this might be not reproducible?

Reply:
true, I am changing this back